docker/cardano/basic_testnet/Dockerfile.cardano-wallet

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:latest
+
+COPY --from=dfordivam/cardano-wallet:2021.9.29 /nix /nix
```

Comment:
The docker image used here could be obtained by the following command in this repo https://github.com/obsidiansystems/cardano-wallet/commits/fb-wallet

`docker load -i $(nix-build -A dockerImage --no-out-link)`

You may push this docker image to official nervos account for use in force-bridge.